Formula/nwchem.rb

```diff
@@ -1,10 +1,10 @@
 class Nwchem < Formula
   desc "NWChem: Open Source High-Performance Computational Chemistry"
   homepage "http://www.nwchem-sw.org"
-  url "https://github.com/nwchemgit/nwchem/releases/download/6.8.1-release/nwchem-6.8.1-release.revision-v6.8-133-ge032219-src.2018-06-14.tar.bz2"
-  version "6.8.1"
-  sha256 "23ce8241a5977a93d8224f66433851c81a08ad58a4c551858ae031485b095ab7"
-  revision 8
+  url "https://github.com/nwchemgit/nwchem/releases/download/v7.0.0-release/nwchem-7.0.0-release.revision-2c9a1c7c-src.2020-02-26.tar.bz2"
+  version "7.0.0"
+  sha256 "1046e13a4c7f95860c8e8fac2b4d80657900ecd07a8242943d564048ce303514"
+  revision 0
```

Comment:
```suggestion
```

Reply:
FYI, if CI returns green I may do this while merging the formula, especially if it takes a long time to build. I'd just wait to see if CI passes before doing anything

Reply:
line removed